data/releasenotes.json

```
@@ -1,5 +1,22 @@
 [
-		{
+	{
+		"version": "v2025-06-20",
+		"date": "2025-06-20",
+		"breaksdb": false,
+		"notes": [
+			"<b>New Feature:</b> Support for JSONB array containment checks: <tt>SELECT '[\"foo\", \"bar\"]'::jsonb @> '\"bar\"'::jsonb;</tt>",
+			"<b>New Feature:</b> Support for <tt>extract(timezone, ..)</tt> for the <tt>timestamptz</tt> type.",
+			"<b>New Feature:</b> Support for the <tt>transaction_timeout</tt> setting.",
+			"<b>PostgreSQL compatibility:</b> <tt>oid</tt>s can now be compared with all integer datatypes.",
+			"<b>PostgreSQL compatibility:</b> Clients may now connect via TLSv1.2 encryption in addition to TLSv1.3.",
+			"<b>PostgreSQL compatibility:</b> Support for subqueries in the <tt>unnest()</tt> function: <tt>select unnest((SELECT ARRAY[1,2]));</tt>.",
+			"<b>PostgreSQL compatibility:</b> Add the postgres-specific legacy types <tt>int2vector</tt> and <tt>oidvector</tt> which predate Postgres's array support and are used by some system tables.",
```

We already had this in a past release, so it is a bit confusing. Maybe something like this:

```suggestion
			"<b>PostgreSQL compatibility:</b> Add server protocol support to the postgres-specific legacy types <tt>int2vector</tt> and <tt>oidvector</tt> in addition to the existing value support.",
```